packages/components/button/src/IconButton/IconButton.tsx

```diff
@@ -32,6 +38,18 @@ interface IconButtonInternalProps
    * Note: 'large' is deprecated
    * */
   size?: ButtonInternalProps['size'];
+
+  /**
+   * Triggers, wether or not to render the tooltip
+   */
+  withTooltip?: boolean;
+
+  /**
+   * A tooltipProps attribute used to conditionally render the tooltip around root element
+   */
+  tooltipProps?: CommonProps &
```

Comment:
Should we have a type check like the one in [Header](https://github.com/contentful/forma-36/blob/a3c52a6d5b4c8a43895ab64324c31427ff39c26e/packages/components/header/src/Header.tsx#L24-L35)?? 
So the editor would complain about a tooltipProps without the withTooltip??